frontend/src/components/MapView.js

```diff
@@ -0,0 +1,46 @@
+import React, { Component } from 'react';
+import { Map, GoogleApiWrapper } from 'google-maps-react';
+import { connect } from "react-redux";
+
+const mapStateToProps = state => {
+  return { articles: state.articles };
+};
+
+const mapStyles = {
+  width: '100%',
+  height: '100%'
+};
+
+class MapView extends Component {
+  constructor() {
+    super();
+
+    this.state = {};
+  }
+
+  render() {
+    return (
+      <div>
+        {this.props.articles.map(article => {
+          return (
+            <Map
+              google={this.props.google}
+              zoom={17}
+              style={mapStyles}
+              initialCenter={{
+                lat: article.location.lat(),
+                lng: article.location.lng()
+              }}
+            />
+          )
+        })}
+      </div>
+    )
+  }
+}
+
+const ConnectedMapView = connect(mapStateToProps);
```

Comment:
can you change the name of the variable to something like connectMapViewToSomething? I  think it will be more helpful to know what the the map is connected to. Does it make sense?